frontend/src/features/properties/filter/PropertyFilter.test.tsx

```diff
@@ -171,7 +170,7 @@ describe('MapFilterBar', () => {
     });
   });
 
-  it('does not display SPP property filter for non-sres users', () => {
+  xit('does not display SPP property filter for non-sres users', () => {
```

Comment:
this test is no longer relevant

Reply:
Might just want to remove it if the functionality no longer requires it